eng/release.yml

```diff
@@ -0,0 +1,200 @@
+trigger: none
+pr: none
+
+parameters:
+- name: IsTestRun
```

Comment:
What is `IsTestRun` for? I'm somewhat confused that there is a `IsDryRun` too.

Reply:
IsTestRun determines if the destination accounts are dotnetcli/dotnetclichecksums vs some test accounts that I set up. IsDryRun determines if the files are actually pushed to those accounts or not.